packages/libs/react-ui/src/components/Form/ActionButtons/NumberFieldActions.tsx

```diff
@@ -0,0 +1,39 @@
+import { MonoAdd, MonoRemove } from '@kadena/react-icons/system';
+import React from 'react';
+import { Button } from '../../Button/Button';
+
+import type { INumberFieldProps } from '../NumberField';
+import { iconSize } from '../NumberField/NumberField.css';
+
+interface INumberFieldActionsProps {
+  isDisabled?: boolean;
+  variant: string;
+  state: any;
+  size?: INumberFieldProps['size'];
+}
+
+export const NumberFieldActions = ({
```

Comment:
nit: I personally wouldn't categorize this as an action button since it should only be used with the number field